app/views/user_case_infos/_form.html.erb

```diff
@@ -63,7 +63,7 @@
     <%= f.label :case_end_date %><br>
     <%= f.datetime_select :case_end_date %>
   </div>
-   <div class="field">
+   <div class="field" hidden>
```

Comment:
Created issue #100. Will accept the hidden field for now in the interest of having things work.